### ISSUE TITLE
uf moves not displaying correctly if they're ub or u or uf

### ISSUE THREAD

**Issue by johngao2** [BKNR/mokujin#2]
I just added `short_input = short_input.replace("u/b or u or u/f", "uf")` to tkfinder and it fixes it, though there might be edge cases where people want the u/b or u version specifically (probably rare though.

**Comment by BKNR** [comment]
If the move is written as `u/b or u or u/f` + button in the framedata, you do get the correct move with uf+button, for example `!alisa uf3` matches to `u/b or u or u/f+3` after `move_simplifier()` as it's currently written. I do admit `!alisa u3` won't find anything, which is less than optimal, but in my experience, usually when people search for framedata on hopkicks, they search `uf3` or  `uf4`.

**Comment by johngao2** [comment]
Please try `!ling aop uf3`, that was the one that gave me an error.

**Comment by BKNR** [comment]
You're right, things out of stances (like AOP) that have "u/b or u or u/f" are not easy to search for, I do have to figure out something for that, that doesn't break other stuff.
